plugins/source.go

```diff
@@ -26,7 +26,7 @@ type SourcePlugin struct {
 	// Tables is all tables supported by this source plugin
 	tables schema.Tables
 	// status sync metrics
-	metrics SourceMetrics
+	metrics *SourceMetrics
```

Comment:
Is this change to a pointer required for the fix?

Reply:
I think yes